### DISCUSSION TITLE
Ned some help

### DISCUSSION THREAD

**Discussion by kevvis** [community/community#29759] [New to GitHub]
Hei. Er ganske ny i denne verdenen, men ønsker virkelig å lære. Håper noen kunne hjulpet meg igang. Jeg holder nå på med termux på Android. Håper noe.har lyst til å vei lede meg.

**Comment by Variable-py** [comment]
If You want to start programming I think python is a Good language for start , you can search on google about it , it has many courses or you can read this documentary at [python.org](https://python.org) , it is very easy language but it's very powerful and useful.

**Comment by jfullstackdev** [comment]
as much as we want to help, but we cannot even translate this

**Comment by queenofcorgis** [comment]
Hi there - this is for users who are new to the world of GitHub, if you have questions about GitHub and its related products ask away here 🚀
